dashboard/test/ui/features/teacher_tools/level_types/bubble_choice.feature

```diff
@@ -10,6 +10,8 @@ Feature: BubbleChoice
     And I click selector ".submitButton"
 
     # Make sure you are taken back to the BubbleChoice activity page with progress
+    And I wait until current URL contains "/lessons/40/levels/1"
+    And I wait for the page to fully load
```

Comment:
Hm, I'm worried this will break the test. `wait for the page to fully load` is a poorly named step that actually [looks for the run button](https://github.com/code-dot-org/code-dot-org/blob/dee6ef102103a33b55a0fa229925ee5c618e363f/dashboard/test/ui/features/step_definitions/steps.rb#L142), which wouldn't appear on a bubble choice page.

Reply:
I'm not sure but maybe. It will at least add a delay before trying to execute jquery, which likely would at least reduce the flakiness.

Looks like `I wait for jquery to load` is also a [step](https://github.com/code-dot-org/code-dot-org/blob/dee6ef102103a33b55a0fa229925ee5c618e363f/dashboard/test/ui/features/step_definitions/steps.rb#L941) so maybe that would help?

Reply:
@bethanyaconnor I removed the `fully load` step and then rant the test from my local machine on test-studio. 
`bundle exec ./runner.rb --html  -c Firefox -f features/teacher_tools/level_types/bubble_choice.feature &`

Without _any_ changes, the test passed 1 out of 3 times.
With just line 13 added (`And I wait until current URL contains "/lessons/40/levels/1"`) the test passed 3 times in a row.

How would you feel about just adding that one new line to see if it helps enough?

Reply:
Either works for me! Especially when we have some evidence that `And I wait until current URL...` reduces flakiness. Sorry this ended up being slightly more complicated than it seemed on the surface :/